website/reviews/listeners.py

```diff
@@ -86,3 +86,47 @@ def reviews_submit_notification_moderators(self, timestamp, context):
                         timestamp,
                         abstract_provider=context['reviewable'].provider,
                         **context)
+
+
+# Handle email notifications to notify moderators of new withdrawal requests
+@reviews_signals.reviews_email_withdrawal_requests.connect
+def reviews_withdrawal_requests_notification(self, timestamp, context):
+    # imports moved here to avoid AppRegistryNotReady error
+    from osf.models import NotificationSubscription
+    from website.profile.utils import get_profile_image_url
+    from website.notifications import emails
+    from website import settings
+
+    # Get NotificationSubscription instance, which contains reference to all subscribers
+    provider_subscription = NotificationSubscription.load(
+        '{}_new_pending_submissions'.format(context['reviewable'].provider._id))
+    preprint = context['reviewable']
+    preprint_word = preprint.provider.preprint_word
+
+    # Set message
+    context['message'] = u'has requested withdrawal of {} "{}".'.format(preprint_word, preprint.node.title)
+    # Set url for profile image of the submitter
+    context['profile_image_url'] = get_profile_image_url(context['requester'])
+    # Set submission url
+    context['reviews_submission_url'] = '{}reviews/preprints/{}/{}'.format(settings.DOMAIN,
+                                                                           preprint.provider._id,
+                                                                           preprint._id)
+    # Store emails to be sent to subscribers instantly (at a 5 min interval)
+    emails.store_emails(provider_subscription.email_transactional.all().values_list('guids___id', flat=True),
+                        'email_transactional',
+                        'new_pending_submissions',
```

Comment:
Initially, I was concerned that this was the wrong template to be using here, but looking into it a bit, it seems almost configurable enough to be useable.

However, there's an issue with using it that I don't see addressed yet -- the `person` `span` isn't guaranteed to return just one person:
```
<span class="person" style="font-weight: bold;">${', '.join(reviewable.node.contributors.values_list('fullname', flat=True))}</span>
```
This could lead to email text along the lines of
```
Alice Allison, Bob Bobson, Carol Carolson [etc.] has requested withdrawal of [...]
```
which is definitely incorrect. Recommended fix: additional `context` var, conditional in the template.